src/yadmark/data/db.py

```diff
@@ -15,3 +15,4 @@ class Observable(Base):
     observables = Column(Text)
     prDIS = Column(Text)
     TargetDIS = Column(Text)
+    NCPositivityCharge = Column(Text)
```

Comment:
This looks like the nth messy config option. I'd start to segment the operator card, and I'm tempted to suggest to do it now

Reply:
and in any case: this is the SQL side which we would not touch - instead we would touch the true opcard

Reply:
Otherwise it's again technical debt

Reply:
> and in any case: this is the SQL side which we would not touch - instead we would touch the true opcard

SQL should reflect the structure of the observable card...

Reply:
> maybe now is the best time to do it in a suitable way.

No - now we implement positivity and get a fit asap. The refactoring is postponed: https://github.com/N3PDF/yadism/issues/147

> SQL should reflect the structure of the observable card...

I disagree, but this discussion is off topic - see above